unlock-app/src/hooks/useRouter.ts

```diff
@@ -0,0 +1,28 @@
+import { useRouter } from 'next/router'
```

Comment:
Rename this file to be correct. It should be `unlock-app/src/hooks/useRouterQueryForLockAddressAndNetworks.ts`